Makefile

```diff
@@ -157,6 +157,18 @@ endif
 		-v `pwd`/output:/workspace/output \
 		gcr.io/vorvan/h2oai/h2o-llmstudio:nightly
 
+.PHONY: docker-stop
+docker-stop:
+	@CONTAINERS=$$(docker ps -a -q --filter ancestor=gcr.io/vorvan/h2oai/h2o-llmstudio:nightly); \
+	if [ -n "$$CONTAINERS" ]; then \
+		docker stop $$CONTAINERS; \
+		docker rm $$CONTAINERS; \
```

Comment:
Maybe remove `docker rm`, or rename the make command to be more explicit that the container will be removed, as well.

Reply:
Actually we don't need to maintain the container, so I remove the `make docker-stop` only adding `make docker-clean-all`